mylyn.tasks/connectors/bugzilla/org.eclipse.mylyn.bugzilla.tests/src/org/eclipse/mylyn/bugzilla/tests/core/BugzillaConfigurationTest.java

```diff
@@ -112,19 +114,12 @@ public void testNullCacheFile() {
 	 * (rdfconfig218.txt) is from mylyn.eclipse.org/bugs218
 	 */
 	public void testRepositoryConfigurationFromFile() throws Exception {
-		BufferedReader inCleaned = null;
-		try {
-			File tempFile = File.createTempFile("XmlCleaner-", "tmp");
-			tempFile.deleteOnExit();
-
-			InputStream stream = BugzillaFixture.getResource("testdata/configuration/rdfconfig218.txt");
-			BufferedReader in = new BufferedReader(new InputStreamReader(stream));
-			try (in) {
-				inCleaned = XmlCleaner.clean(in, tempFile);
-				if (tempFile != null) {
-					tempFile.delete();
```

Comment:
I'm not sure why this statement was needed right here, but in refactored version it is moved to another location for some reason.

Reply:
It made more sense to move the delete() to the final block